v2/pink-sb/src/lib/accordion-table/cell/Cell.svelte

```diff
@@ -0,0 +1,46 @@
+<script lang="ts">
+    import Icon from '$lib/Icon.svelte';
+    import type { RootProp } from '../index.js';
+    import Stack from '$lib/layout/Stack.svelte';
+    import { type ComponentProps, getContext } from 'svelte';
+    import { IconChevronDown } from '@appwrite.io/pink-icons-svelte';
+
+    export let root: RootProp;
+    export let column: string;
+
+    let expandable: boolean;
+    $: expandable = getContext<boolean>('expandable');
```

Comment:
i dont think we need the explicit type in this case
```suggestion
    $: expandable = getContext<boolean>('expandable');
```